src/System.Net.HttpListener/tests/HttpRequestStreamTests.cs

```diff
@@ -313,7 +313,6 @@ public void Dispose()
         }
 
         [Fact]
-        [ActiveIssue(18128, platforms: TestPlatforms.AnyUnix)] // NotSupportedException thrown instead of InvalidOperationException
```

Comment:
I think that issue was fixed in https://github.com/dotnet/corefx/pull/18450/commits/5858c5b4b3d438d17aec8090906a62a9e87269e7
I just undisabled the test too early!